docs/application/native/get-started/iot/first-app.md

```diff
@@ -0,0 +1,292 @@
+# Create Your First Tizen IoT Native Service Application
+
+**Welcome to Tizen IoT Native Service Application development!**
+
+> [!NOTE]
+> Service applications are Tizen native applications which run in the background with no graphical user interface.
+> These applications can be very useful for activities, such as getting sensor data in the background that needs to run periodically or continuously but does not require any user interaction.
+
+An IoT native service application can be created using the C language and run on Raspberry Pi. The application uses the native APIs, which provide various interfaces to the device hardware allowing you to take advantage of numerous capabilities tailored to run with limited device resources.
+
+You can study the following contents to help familiarize yourself with the Tizen [Native application development process](../../tutorials/process/app-dev-process.md) as well as using Tizen Studio and deploying the created application on Raspberry Pi. With the instructions, you can create and run a basic IoT native service application, which displays some text on the log with no user interaction:
```

Comment:
I unresolve this conversation back as the issued sentence is not updated yet. If we resolve this conversation it can be confused to be a comment that already applied.

@surbhig12 Would you please check if you comment to change to "Study the following instructions to help familiarize yourself.." is applied?

**Related :** https://github.com/Samsung/tizen-docs/pull/1198#discussion_r470075234

Reply:
@ey928-lee, please incorporate the comment:

Study the following instructions to help familiarize yourself with the Tizen [Native application development process](../../tutorials/process/app-dev-process.md) as well as using Tizen Studio and deploying the created application on Raspberry Pi. With the instructions, you can create and run a basic IoT native service application, which displays some text on the log with no user interaction:

Reply:
Okay. I will incorporate the comment.